### PR TITLE
Updating a few things

### PR DESCRIPTION
**Pull request by flamableassassin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,13 +2,21 @@
...
     "url": "git+https://github.com/project-blurple/blurple-hammer.git"
   },
   "author": "Promise <me@promise.solutions> (https://promise.solutions/)",
+  "contributors": [
+    "Promise <me@promise.solutions> (https://promise.solutions/)",
+    "Gigabitzz <> (https://www.gigabitz.pw)",
+    "wrac4242 <> ()",
+    "flamableassassin <lighter@highlyflammable.tech> (https://highlyflammable.tech/)"
+   ],
+
```
Comment (promise) [inline package.json:19]: Contributors are shown here: https://github.com/project-blurple/blurple-hammer/graphs/contributors
Will not keep this up-to-date. Please remove before I can accept the PR.

File: src/commands/eval.js [hunk 2]
```diff
@@ -1,28 +1,26 @@
...
+module.exports.run = async ({ channel, client }, { code }) => {
   try {
-    let evaled = eval(code);
+    let evaled = await eval(code);
     if (typeof evaled != "string") evaled = require("util").inspect(evaled);
+    evaled = evaled.replace(client.token, "-- Redacted ---");
+
     channel.send(`🆗 Evaluated successfully.\n\`\`\`js\n${evaled}\`\`\``);
-  } catch(e) {
+  } catch (e) {
```
Comment (promise) [inline src/commands/eval.js:20]: ?

File: src/commands/eval.js [hunk 3]
```diff
@@ -1,28 +1,26 @@
...
 };
 
-module.exports.run = async ({ channel }, { code }) => {
+module.exports.run = async ({ channel, client }, { code }) => {
   try {
-    let evaled = eval(code);
+    let evaled = await eval(code);
     if (typeof evaled != "string") evaled = require("util").inspect(evaled);
+    evaled = evaled.replace(client.token, "-- Redacted ---");
+
```
Comment (promise) [inline src/commands/eval.js:18]: If I ever do leak the client token then I will simply refresh it. I actually fetch the client token through this command quite a lot in private channels, so I would like to keep this untouched.

File: src/commands/eval.js [hunk 4]
```diff
@@ -1,28 +1,26 @@
...
-      description: "The code you want to run through the bot.",
-      required: true
-    }
-  ],
+  options: [{
+    type: 3,
+    name: "code",
+    description: "The code you want to run through the bot.",
+    required: true
+  }],
```
Comment (promise) [inline src/commands/eval.js:8]: Would like to keep this as-is.

**Review by promise — CHANGES_REQUESTED** [review]

File: src/commands/eval.js (hunk 2 above)
Comment (flamableassassin) [reply src/commands/eval.js:20]: that was done by my beautifier

File: src/commands/eval.js (hunk 4 above)
Comment (flamableassassin) [reply src/commands/eval.js:8]: 👍 ok i shall change it back

File: package.json (hunk 1 above)
Comment (flamableassassin) [reply package.json:19]: 👍 i shall remove it

File: src/commands/eval.js (hunk 3 above)
Comment (flamableassassin) [reply src/commands/eval.js:18]: I added because it can be leaked without realising because its attached to the client class. I shall remove it tho

**Review by promise — APPROVED** [review]

**Pull request merged**